Tests/LibWeb/TestConfig.ini

```diff
@@ -165,14 +161,5 @@ Text/input/wpt-import/css/css-backgrounds/animations/discrete-no-interpolation.h
 ; https://github.com/LadybirdBrowser/ladybird/issues/2314
 Text/input/test-http-test-server.html
 
-; Too slow for CI
-Text/input/wpt-import/dom/nodes/ParentNode-querySelector-All.html
-Text/input/wpt-import/html/rendering/replaced-elements/svg-embedded-sizing/svg-in-object-fixed.html
-Text/input/wpt-import/html/rendering/replaced-elements/svg-embedded-sizing/svg-in-object-auto.html
-Text/input/wpt-import/html/rendering/replaced-elements/svg-embedded-sizing/svg-in-object-percentage.html
-Text/input/wpt-import/html/rendering/replaced-elements/svg-embedded-sizing/svg-in-iframe-fixed.html
-Text/input/wpt-import/html/rendering/replaced-elements/svg-embedded-sizing/svg-in-iframe-auto.html
-Text/input/wpt-import/html/rendering/replaced-elements/svg-embedded-sizing/svg-in-iframe-percentage.html
-
 ; https://github.com/LadybirdBrowser/ladybird/issues/2659
 Text/input/wpt-import/css/cssom/serialize-values.html
```

Comment:
The serialize-values test was also disabled because of timeouts, and can probably be re-enabled.

Reply:
FYI this was the test I was referencing here: https://github.com/LadybirdBrowser/ladybird/issues/2720
If someone runs that test in isolation, they're likely to hit that crash.

Reply:
:man_facepalming: I don't know why I forgot that. Fixing the crash is at the top of my list though.